app/Controllers/entryController.php

```diff
@@ -117,11 +117,11 @@ public function readAction(): void {
 
 		if (!$this->ajax) {
 			Minz_Request::good($is_read ? _t('feedback.sub.articles.marked_read') : _t('feedback.sub.articles.marked_unread'),
-			array(
+			[
 				'c' => 'index',
 				'a' => 'index',
 				'params' => $params,
-			));
+				]);
```

Comment:
Strange indenting of the closing part of the array in many of your changes here and below

Reply:
There were still many arrays with strange indenting. Fixed in https://github.com/FreshRSS/FreshRSS/pull/5504/commits/a13685f4ed20537e4fccd4b603ee1e45d9e516ce